feature-utils/poly-look/test/react-components/filterChips.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import FilterChips from "../../src/react-components/filterChips/filterChips.jsx";
+
+const chipsContent = [...Array(3).keys()].map((i) => `chip${i}`);
+const mockedHandleClick = jest.fn();
+const allChipContent = "All";
+// const mockedActiveChips = [chipsContent[1], chipsContent[2]];
+
+const component = (
+  <FilterChips
+    chipsContent={chipsContent}
+    onChipClick={mockedHandleClick}
+    allChip={{ translation: allChipContent }}
+  />
+);
+
+it("Chips present in document", () => {
+  const { getByText } = render(component);
+  for (const chipId of chipsContent) {
+    const chipElement = getByText(chipId);
+    expect(chipElement).toBeInTheDocument();
+  }
+});
+
+it("Checks onChipClick is called", () => {
+  const { getByText } = render(component);
+  for (const chipId of chipsContent) {
+    const chipElement = getByText(chipId);
+    fireEvent.click(chipElement, mockedHandleClick);
+    expect(mockedHandleClick).toHaveBeenCalledWith(chipId, [chipId]);
+  }
+});
+
+it("There is a Chip that selects all elements", () => {
+  const { getAllByText } = render(component);
+  const allChipElement = getAllByText(allChipContent);
+  const allChipElementLabel = Object.values(allChipElement[0])[1].children;
+  expect(component.props.allChip).toBeTruthy();
+  expect(allChipElementLabel).toMatch(allChipContent);
```

Comment:
This basically just checks that this chip exists - which is cool, but what about what it _does_? Meaning, clicking it makes _all_ chips active?